microcosm/metadata.py

```diff
@@ -23,7 +23,8 @@ def __init__(self, name, debug=False, testing=False, import_name=None, root_path
         self.name = name
         self.debug = debug
         self.testing = testing
-        self.root_path = root_path or self.get_root_path(import_name or name)
+        self.import_name = import_name or self.name
+        self.root_path = root_path or self.get_root_path(self.import_name)
```

Comment:
Unrelated change; I want this import value for Flask application initialization.